src/main/java/build/buildfarm/instance/shard/ShardInstance.java

```diff
@@ -1651,78 +1653,126 @@ public void onFailure(Throwable t) {
     return requeuedFuture;
   }
 
+  String operationBlockedError(String operationName) {
+    return String.format("Operation %s not requeued. " + BLOCK_LIST_ERROR, operationName);
```

Comment:
BLOCK_LIST_ERROR is used in other places so I'm going to make the messages all constants as you suggest.  I still need to use .format() to fill in the context information.

Reply:
Yeah, they are constants for good reasons. Since we are building a string here, maybe we can try something like 
```
String.format("Operation %s not requeued. %s", operationName, BLOCK_LIST_ERROR);
```
But yeah, this is no big deal at all.

Reply:
Thanks for clarifying.  Hopefully the new constants are equally clean